qiskit_experiments/base_experiment.py

```diff
@@ -77,7 +77,20 @@ def __init__(self, qubits: Iterable[int], experiment_type: Optional[str] = None)
         self._analysis_options = self._default_analysis_options()
 
         # Set initial layout from qubits
-        self._transpile_options.initial_layout = self._physical_qubits
+        self._transpile_options.initial_layout = list(self._physical_qubits)
+
+    def metadata(self):
+        """Return experiment metadata."""
+        metadata = {
```

Comment:
backend is not stored in this metadata?

Reply:
Some metadata is already stored in the (result DB) ExperimentData including backend, experiment type, tags, so this is field intended for additional things that aren't part of the base (terra) class.